tests/unit/transformers/test_categorical.py

```diff
@@ -1856,3 +1857,70 @@ def test__reverse_transform_add_noise(self):
 
         # Assert
         pd.testing.assert_series_equal(out, pd.Series(['a', 'b', 'c']))
+
+
+class TestCustomLabelEncoder:
+
+    def test___init__(self):
+        """Passed arguments must be stored as attributes."""
```

Comment:
nit: Should we follow the convention of starting the description with "Test the ``__init__`` method"?